### PR TITLE
Sql queries

### PR DESCRIPTION
**Pull request by Ochiengsteven**
📊 **SQL Queries and Results**

This pull request includes the setup of the `vet_clinic` database, data insertion into the `animals` table, and SQL queries for various tasks based on the project requirements.

**Database Setup:**

- Created a database named `vet_clinic`.
- Defined a table named `animals` with the specified columns in the database.

**Data Insertion:**

- Inserted data for four animals into the `animals` table.

**SQL Queries:**

1. Find all animals whose name ends in "mon."
2. List the name of all animals born between 2016 and 2019.
3. List the name of all animals that are neutered and have less than 3 escape attempts.
4. List the date of birth of all animals named either "Agumon" or "Pikachu."
5. List name and escape attempts of animals that weigh more than 10.5kg.
6. Find all animals that are neutered.
7. Find all animals not named Gabumon.
8. Find all animals with a weight between 10.4kg and 17.3kg (inclusive).

## 📷 Screenshot <a name="screenshot"></a>

Screenshot

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -0,0 +1,23 @@
+-- Find all animals whose name ends in "mon"
+SELECT * FROM animals WHERE name LIKE '%mon';
+
+-- List the name of all animals born between 2016 and 2019
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
```
Comment (V-Blaze) [inline queries.sql:5]: - You have done great so far :+1: But there are a few changes you need to make before moving to the next milestone. :+1: :+1: 

> Take a screenshot of the results of your queries.
Remember to add the screenshot with the results of your queries to your Pull Request description.

- [ ] According to the project requirement, you were supposed to take screenshots of all these queries and include them in the pull request description, currently, I can you have a screenshot in a .png format, I'll suggest you also include the screenshot in the pull request description as required and for easy viewing. :+1: :smile:

File: README.md [hunk 2]
```diff
@@ -1 +1,279 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p> -->
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (V-Blaze) [inline README.md]: - [ ] Your MIT LICENSE link leads to a 404 error page, a broken link makes the readme look a bit unprofessional. I'll suggest you create a License file and link this correctly. :+1: :+1: 
- - Screenshot reference:
Screenshot from 2023-09-18 11-42-12

**Review by V-Blaze — CHANGES_REQUESTED** [review]
## Hi @Ochiengsteven 

## Required changes 🖊️ 
# Almost there 🟡 🟦 🟨 

almost-there


Great work so far, You have come a long way, keep moving forward👏 👍  
You've done well implementing the project requirements, but there are still some issues I think you need to improve on. 😄 👋 


 ## Highlights
- Everything that creates the table structure is in the schema.sql file.✔
-  Queries written correctly👋 
-  Data are correctly inserted into the table✔

 ## Required Changes ♻️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (Ochiengsteven) [reply queries.sql:5]: Hello @V-Blaze I've tried adding the screenshot in the PR and its not showing. Regarding the licence link I've updated it

**Review by julie-ify — APPROVED** [review]
Hi @Ochiengsteven,

## Highlights!
- Good code organization and working queries ✔️
- Descriptive README ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**